sopel/coretasks.py

```diff
@@ -41,6 +49,11 @@
 
 
 def setup(bot):
+    """Setup the coretasks plugin.
```

Comment:
Per [setup (v.)](https://en.wiktionary.org/wiki/setup#Verb):

```suggestion
    """Set up the coretasks plugin.
```